config/katello.migrations/171005163313-empty-proxy-strings.rb

```diff
@@ -0,0 +1,8 @@
+if answers['katello']
+  ['url', 'port', 'username', 'password'].each do |name|
+    key = "proxy_#{name}"
+    if answers['katello'][key] == ''
```

Comment:
Ahh, and so after this point '' will never be a valid string again? Unless I as a user go edit the answer file directly.

Reply:
Currently the katello module refuses to use them so this clears them which used to be the same. If we do start accepting empty strings again we can remove this migration.